File: TD6V2/app/src/main/java/com/example/td6_v2/MainActivity.java
package com.example.td6_v2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button goListRepo = findViewById(R.id.goListRepo);
        goListRepo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,ListRepos.class));
            }
        });

        Button goSearchRepo = findViewById(R.id.goSearchRepo);
        goSearchRepo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,SearchRepo.class));
            }
        });

    }
}

File: TD2/app/src/main/java/com/example/td2/MainActivity.kt
package com.example.td2

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val buttonMeteo = findViewById<Button>(R.id.buttonMeteo);
        val buttonRepas = findViewById<Button>(R.id.buttonRepas);

        buttonMeteo.setOnClickListener {
            startActivity(  Intent(this,Meteo_splash::class.java))
        }
        buttonRepas.setOnClickListener {
            startActivity(Intent(this,Repas::class.java))
        }


    }
};